nuget/Opc.Ua.Symbols.nuspec

```diff
@@ -14,6 +14,7 @@
     <tags>OPCFoundation OPC-UA netstandard ios linux dotnet net netcore uwp</tags>
     <dependencies>
       <dependency id="OPCFoundation.NetStandard.Opc.Ua.Core.Debug" version="$version$"/>
+      <dependency id="OPCFoundation.NetStandard.Opc.Ua.Bindings.Https" version="$version$"/>
```

Comment:
Shouldn't be "_Ua.Bindings.Https.Debug_" here?

Reply:
yep, good catch!